### PR TITLE
Fix cypress folder path

### PR DESCRIPTION
**Pull request by sinansonmez**
fix cypress folder path

Screen Shot 2023-03-23 at 23 46 00

### CODE REVIEW

**Comment by sinansonmez** [comment]
cc: @mickmister

**Comment by mickmister** [comment]
Thanks @sinansonmez! Can you also make this change to the `Playwright` section below the `Cypress`?

**Comment by sinansonmez** [comment]
Done @mickmister

File: .gitpod.yml [hunk 1]
```diff
@@ -82,12 +82,12 @@ tasks:
...
   - name: Cypress
     init: |
-      cd mattermost-server/webapp/e2e/cypress
+      cd ../mattermost-server/e2e/cypress
       npm i
 
   - name: Playwright
     init: |
-      cd mattermost-server/webapp/e2e/cypress
+      cd ../mattermost-server/e2e/cypress
```
Comment (mickmister) [inline .gitpod.yml]: Looks like I goofed here in two separate ways

```suggestion
      cd ../mattermost-server/e2e/playwright
```

**Comment by mickmister** [comment]
LGTM now :+1:

**Pull request merged**